webapp/www/index.html

```diff
@@ -46,6 +46,7 @@
       <ion-nav-buttons side="primary">
         <a class="button button-block button-calm icon-left ion-home" ui-sref="home">Home</a>
         <a class="button button-block button-calm icon-left ion-ios-analytics" ui-sref="tab.heatmap">Analysis</a>
+        <a class="button button-block button-calm icon-left ion-map" href="http://54.162.140.55:8080">Trip Planning</a>
```

Comment:
This is not strictly correct. Note that for the heatmap and home tabs, the tab header is visible at the top, so if you are in heatmap, you can go back to home. But with this option, if you click on trip planner, you go to the trip planner page but you can't go to (say) heatmap without clicking the back button.

You really do need to create a new tab for the trip planner, but it should be fairly straightforward to have it load the OTP server page as the HTML.